topdown/crypto.go

```diff
@@ -56,7 +56,23 @@ func builtinCryptoX509ParseCertificates(_ BuiltinContext, operands []*ast.Term,
 		return err
 	}
 
-	v, err := ast.InterfaceToValue(certs)
+	// add a field to certs containing the URIs as strings
+	processedCerts := make([]struct {
+		x509.Certificate
+		RawURIs []string `json:"RawURIs"`
+	}, len(certs))
+
+	for i, cert := range certs {
+		processedCerts[i].Certificate = *cert
+		if cert.URIs != nil {
+			processedCerts[i].RawURIs = make([]string, len(cert.URIs))
+			for j, uri := range cert.URIs {
+				processedCerts[i].RawURIs[j] = uri.String()
+			}
+		}
+	}
+
+	v, err := ast.InterfaceToValue(processedCerts)
```

Comment:
We'd probably need something similar for `crypto.x509.parse_and_verify_certificates` ?

Reply:
Good catch, I've updated the implementation of this too in f71330c342ab1de9a5489caad365eac3eccf2e02